html/modules/custom/hr_paragraphs/src/Controller/InformKeyFiguresController.php

```diff
@@ -21,7 +21,7 @@ class InformKeyFiguresController extends BaseKeyFiguresController {
   public function __construct(ClientInterface $http_client, CacheBackendInterface $cache) {
     parent::__construct($http_client, $cache);
 
-    $this->apiUrl .= 'inform/';
+    $this->apiUrl .= 'inform-risk/';
```

Comment:
this was the only change in the branch yesterday. Still the case once we merge dev?

Reply:
```
Updating ea37d1e..9662a78
Fast-forward
 html/modules/custom/hr_paragraphs/src/Controller/InformKeyFiguresController.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```